.travis.yml

```diff
@@ -7,16 +7,15 @@ python:
   - 2.7
   - 3.6
   - 3.7-dev
+
 services:
   - docker
 
+before_install:
+  - sudo apt-get install -y
+
 install:
-  - pip install tox tox-travis molecule docker
+  - pip install tox tox-travis
 
 script:
   - tox
-  - molecule --version
-  - ansible --version
-  - molecule lint
-  - molecule syntax
-  - molecule test
```

Comment:
The two blocks above seem to revert parts of b83d15f09b91fe92cde761f91f154fc4f1b13eb3, can you squash those two commits to minimize the diffs and reduce the chance of conflicts, please?

Reply:
I moved the changes to .travis.yml in one squashed commit.